### ISSUE TITLE
Loss and training

### ISSUE THREAD

**Issue by Jeanselme** [SherlockLiao/pytorch-beginner#11]
Hi, I think that there is a few mistakes in the simple and convolutional autoencoders :
- The displayed loss is the loss on the last image of the epoch instead of the loss over the whole epoch
- The autoencoder is not tested on the test dataset
- The autoencoder is never in "train" or "test" mode
Is it normal ?
